server/server/src/sync_on_central/mod.rs

```diff
@@ -14,17 +14,17 @@ use service::{
     settings::Settings,
     sync::{
         api_v6::{
-            SiteStatusRequestV6, SiteStatusResponseV6, SyncDownloadFileRequestV6, SyncParsedErrorV6, SyncPullRequestV6, SyncPullResponseV6,
-            SyncPushRequestV6, SyncPushResponseV6, SyncUploadFileRequestV6,
-            SyncUploadFileResponseV6,
+            SiteStatusRequestV6, SiteStatusResponseV6, SyncDownloadFileRequestV6,
+            SyncParsedErrorV6, SyncPullRequestV6, SyncPullResponseV6, SyncPushRequestV6,
+            SyncPushResponseV6, SyncUploadFileRequestV6, SyncUploadFileResponseV6,
         },
         sync_on_central,
     },
 };
 
 pub fn config_sync_on_central(cfg: &mut web::ServiceConfig) {
     cfg.service(
-        web::scope("central")
+        web::scope("central/sync")
```

Comment:
Change the `/sync` scope to be set here, so `/central/sync` requests match on this service, other `/central` requests go to the new `/central` service